azure-pipelines.yml

```diff
@@ -0,0 +1,82 @@
+parameters:
+  - name: terraformVersion
+    type: string
+    default: "0.13.4"
+  - name: sendgridEnvironments
+    type: object
+    default:
+    - environment: nonprod
+      serviceConnection: dts-management-nonprod-intsvc
+      rg: mgmt-state-store-nonprod
+      location: uksouth
+      storage_account: dtsmgmtstatestorenonprod
+      container: mgmtstatestorecontainernonprod
+
+    - environment: prod
+      serviceConnection: dts-management-prod-intsvc
+      rg: mgmt-state-store-prod
+      location: uksouth
+      storage_account: dtsmgmtstatestoreprod
+      container: mgmtstatestorecontainerprod
+
+variables:
+  isMain: $[eq(variables['Build.SourceBranch'], 'refs/heads/master')]
+
+pr:
+- Integration
+
+pool:
+  name: Azure Pipelines
+  demands: azureps
+  vmImage: 'windows-latest'
```

Comment:
windows??? should be using ubuntu

Reply:
carried over from existing pipeline - now removed.